app/helpers/record_helper.rb

```diff
@@ -1,4 +1,10 @@
 module RecordHelper
+  # Campaigns/Campaign/ShortName comes from ECHO10
+  # Projects/ShortName comes from UMM-JSON
+  # Project/Short_Name comes from DIF10
+  CAMPAIGN_COLUMNS = %w[Campaigns/Campaign/ShortName
+                        Projects/ShortName
+                        Project/Short_Name]
```

Comment:
look in record_formats/dif10_record.rb, record_formats/echo10_record.rb, record_formats/umm_record.rb, I feel like the logic for pulling out the right column name for campaign should be in there and you can just say:
record.campaign_column
This has the benefit if a new format is added, it will be apparent what needs to be implemented for that new format.

Reply:
Doing it that way means inspecting every record after the DB query.  Doing it this way means the DB does that work for us.  I would prefer not to change it, but I should find some place to put comments to try to make this more maintainable.